Improve "Creating Forms" docs (#206)
yiisoft/form: docs/creating-forms.md

# Creating Forms

> A form, that is displayed on the client-side, will in most cases have a corresponding form model which is used to
> [validate input](https://github.com/yiisoft/validator).

Reviewer: ```suggestion
A form displayed on the client-side usually has a corresponding form model which is used to
[validate input](https://github.com/yiisoft/validator).
```
